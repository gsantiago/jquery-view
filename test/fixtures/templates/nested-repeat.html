<ol>
  <li :repeat="user in users">
    {{user.name}}
    <span class="skill" :repeat="skill in user.skills">
      {{skill}}
    </span>
  </li>
</ol>