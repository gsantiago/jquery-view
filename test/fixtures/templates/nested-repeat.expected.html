<ol>
  <li>
    Guilherme
    <span class="skill">
      html
    </span><span class="skill">
      css
    </span><span class="skill">
      js
    </span>
  </li><li>
    Zilla
    <span class="skill">
      php
    </span><span class="skill">
      mysql
    </span>
  </li><li>
    Junin
  </li>
</ol>